platform/net/ubuntu_net_manager_test.go

```diff
@@ -904,6 +938,108 @@ iface ethstatic inet static
 				Expect(networkConfig.StringContents()).To(Equal(expectedNetworkConfiguration))
 			})
 		})
+
+		Context("when manual networks were configured with portal IP", func() {
+			var (
+				portalNetwork  boshsettings.Network
+				staticNetwork  boshsettings.Network
+				staticNetwork1 boshsettings.Network
+			)
+			BeforeEach(func() {
+				interfaceAddrsProvider.GetInterfaceAddresses = []boship.InterfaceAddress{
+					boship.NewSimpleInterfaceAddress("ethstatic", "1.2.3.5"),
+				}
+				portalNetwork = boshsettings.Network{
+					Type:     "manual",
+					IP:       "10.112.166.136",
+					Netmask:  "255.255.255.192",
+					Resolved: false,
+					UseDHCP:  false,
+					DNS:      []string{"8.8.8.8"},
+					Alias:    "eth0:0",
+				}
+				staticNetwork = boshsettings.Network{
+					Type:     "dynamic",
+					IP:       "169.50.68.75",
+					Netmask:  "255.255.255.224",
+					Gateway:  "169.50.68.65",
+					Default:  []string{"gateway", "dns"},
+					Resolved: false,
+					UseDHCP:  false,
+					DNS:      []string{"8.8.8.8", "10.0.80.11", "10.0.80.12"},
+					Mac:      "06:64:d4:7d:63:71",
+					Alias:    "eth1",
+				}
+				staticNetwork1 = boshsettings.Network{
+					Type:     "dynamic",
+					IP:       "10.112.39.113",
+					Netmask:  "255.255.255.128",
+					Resolved: false,
+					UseDHCP:  false,
+					DNS:      []string{"8.8.8.8", "10.0.80.11", "10.0.80.12"},
+					Mac:      "06:b7:e8:0c:38:d8",
+					Alias:    "eth0",
+				}
+				interfaceAddrsProvider.GetInterfaceAddresses = []boship.InterfaceAddress{
+					boship.NewSimpleInterfaceAddress("eth0", "10.112.39.113"),
+					boship.NewSimpleInterfaceAddress("eth1", "169.50.68.75"),
+				}
+				fs.WriteFileString("/etc/resolv.conf", `
+nameserver 8.8.8.8
+nameserver 10.0.80.11
+nameserver 10.0.80.12
+`)
+			})
+
+			It("succeeds", func() {
```

Comment:
What happens when it fails? There are a few error conditions to consider? We should have tests for these, no?